tools/cli/workflow_commands.go

```diff
@@ -2312,10 +2312,7 @@ func getFirstDecisionTaskByType(
 			break
 		}
 	}
-	if decisionFinishID == 0 {
-		return 0, printErrorAndReturn("Get DecisionFinishID failed", fmt.Errorf("no DecisionFinishID"))
-	}
-	return
+	return 0, printErrorAndReturn("Get DecisionFinishID failed", fmt.Errorf("no DecisionFinishID"))
```

Comment:
Would you mind add some comments on why decisionFinishID will be zero here

Reply:
Yeah, if it finds any matched decision type, it returns directly in the code above. So the decisionFinishID will always be the default value 0 when it reaches here. I will change it this line to return decisionFinishID, error which will be more clear

Reply:
Sounds good, thanks